sdk/ml/azure-ai-ml/.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+files: sdk/ml/
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.3.0 # Needs to be updated whenever azure sdk team bumps
+    hooks:
+    -   id: black
+        args: [--config=eng/black-pyproject.toml]
```

Comment:
@kdestin Does the args input here need to change to ../../eng/black-pyproject.toml since the pre-commit yaml isn't being defined in the repo root?

Reply:
git hooks always run relative to the repo root, so paths should be written relative to the repo root and not the config file location.